app/components/status_tags/ownership_certificate_component.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module StatusTags
+  class OwnershipCertificateComponent < StatusTags::BaseComponent
+    def initialize(planning_application:)
+      @planning_application = planning_application
+    end
+
+    private
+
+    attr_reader :planning_application
+
+    def status
+      if planning_application.valid_ownership_certificate.nil?
```

Comment:
I think we could do this instead:

```suggestion
      if planning_application.ownership_certificate.nil?
```

Then that would allow us to avoid a nullable boolean and keep Rubocop happy.